lib/less-rhino/index.js

```diff
@@ -104,16 +104,16 @@ less.Parser.fileLoader = function (file, currentFileInfo, callback, env) {
         newFileInfo.entryPath = currentFileInfo.entryPath;
         newFileInfo.rootpath = currentFileInfo.rootpath;
         newFileInfo.rootFilename = currentFileInfo.rootFilename;
-        newFileInfo.relativeUrls = currentFileInfo.relativeUrls;
+        newFileInfo.rewriteUrls = currentFileInfo.rewriteUrls;
     } else {
         newFileInfo.entryPath = path;
         newFileInfo.rootpath = less.rootpath || path;
         newFileInfo.rootFilename = href;
-        newFileInfo.relativeUrls = env.relativeUrls;
+        newFileInfo.rewriteUrls = env.rewriteUrls;
     }
 
     var j = file.lastIndexOf('/');
-    if (newFileInfo.relativeUrls && !/^(?:[a-z-]+:|\/)/.test(file) && j != -1) {
+    if (newFileInfo.rewriteUrls && !/^(?:[a-z-]+:|\/)/.test(file) && j != -1) {
```

Comment:
Not sure if we need to change that part. However, for me it looks like `less.Parser.fileLoader` is not used anywhere. Is that file even up-to-date?

Reply:
No, `less-rhino` should have been pulled in 3.0, but I just wasn't sure. I doubt it even still works. Ideally, PR #2985 should be re-written and pulled into its own repo with `less` as a dependency (as in `https://github.com/less/less-java`), and then `less-rhino` should be removed as a sub-folder.

Long and short, there aren't any active Java maintainers in the Less core team, so we really can't manage it within this repo.